html/changelogs/archive/2021-08.yml

```diff
@@ -569,7 +569,7 @@
   Lewdcifer:
   - balance: Reverts Powerfist Stun.
   Losenis:
-  - balance: T-29 Smartmachinegun can now mount miniscope
+  - balance: SG-29 Smartmachinegun can now mount miniscope
```

Comment:
lmao please don't change any *.ymls
these are for referencing, doesn't impact the actual game

Reply:
Oops, that's what I get for using replace all

Reply:
It might be for the best if people want to look back though, since in the future it wouldn't be called the T-29 and if they wanted to check the weapon history they'd have difficulty finding it.